src/js/constants.js

```diff
@@ -37,6 +39,12 @@ export const COMMAND_TYPE_ANALOG = 'analog';
 
 export const COMMAND_TYPE_OBIS_OBSERVER = 'obisObserver';
 
+export const COMMAND_TYPE_MTX = 'mtx';
+
+export const ACCESS_KEY_LENGTH_BYTES = 16;
+
+export const DEFAULT_ACCESS_KEY = utils.getHexFromBytes(new Uint8Array([...Array(16).keys()]), {separator: ' '});
```

Comment:
how about `DEFAULT_ACCESS_KEY` and `DEFAULT_ACCESS_KEY_HEX` ?

Reply:
After a discussion, we've decided that an additional constant is not necessary.